pygmt/src/grd2cpt.py

```diff
@@ -159,6 +161,14 @@ def grd2cpt(grid, **kwargs):
         range. Note that ``cyclic=True`` cannot be set together with
         ``categorical=True``.
     {verbose}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 30m grid with "gridline" registration
+    >>> grid = pygmt.datasets.load_earth_relief("30m", registration="gridline")
+    >>> # create a CPT with grd2cpt
+    >>> cpt = pygmt.grd2cpt(grid=grid)
```

Comment:
Currently, `grd2cpt` doesn't return a CPT object. It simply generate a hidden, current CPT.

Reply:
I did not realize that; I changed the example